ingest/ingest_pipeline.py

```diff
@@ -400,16 +399,26 @@ def subsample(self):
 
         if self.cell_metadata_file is not None:
             try:
-                subsample.prepare_cell_metadata()
-                for data in subsample.subsample("study"):
-                    load_status = self.load_subsample(
-                        Clusters.COLLECTION_NAME,
-                        data,
-                        subsample.set_data_array,
-                        "study",
+                # Annotation names for cluster file
+                annot_names = SubSample.get_annot_names(subsample.annot_column_headers)
```

Comment:
I don't understand what annotation names have to do with checking unique cells.  the 'annotation names' are the cluster file column headers, (e.g. 'Intensity', 'subgroup').  Do you mean cluser_file_cell_names here?

Reply:
This was actually a problem with context switching. I was just working on expression files where the cell names are the first lines. Made the update in d2979c8